src/components/Property/Types/NumericProperty.tsx

```diff
@@ -0,0 +1,117 @@
+import { useState } from 'react';
+import { MdOutlineEdit } from 'react-icons/md';
```

Comment:
re-export from `icons/icons.tsx` and remove `Md` part of the re-import for consistency